SystemInformer/modlist.c

```diff
@@ -86,7 +86,7 @@ VOID PhInitializeModuleList(
 
     // Default columns
     PhAddTreeNewColumn(Context->TreeNewHandle, PHMOTLC_NAME, TRUE, L"Name", 100, PH_ALIGN_LEFT, -2, 0);
-    PhAddTreeNewColumn(Context->TreeNewHandle, PHMOTLC_BASEADDRESS, TRUE, L"Base address", 80, PH_ALIGN_RIGHT, 0, DT_RIGHT);
+    PhAddTreeNewColumn(Context->TreeNewHandle, PHMOTLC_BASEADDRESS, TRUE, L"Base address", 80, PH_ALIGN_RIGHT | PH_ALIGN_MONOSPACE_FONT, 0, DT_RIGHT);
```

Comment:
There is also a few other locations in `modlist.c` and `modprv.c` that needs updated to use the new formatting:

https://github.com/winsiderss/systeminformer/blob/master/SystemInformer/modprv.c#L647
https://github.com/winsiderss/systeminformer/blob/master/SystemInformer/modlist.c#L1034-L1047

https://github.com/winsiderss/systeminformer/blob/master/SystemInformer/modlist.c#L108-L109

I thought about just changing `PhPrintPointer` but I'm hesitant since it's used in some other places too. Let's just focus on these locations since we're touching the `Modules` and `Memory` tabs of the process information.

Reply:
Yeah, it'd be definitely be convenient if `PhPrintPointer` did handle it since I feel a bit bad about copy-pasting the same code around.